frontend/packages/dev-console/src/components/import/deployImage-submit-utils.ts

```diff
@@ -287,7 +287,14 @@ export const createResources = (
   } else if (!dryRun) {
     // Do not run serverless call during the dry run.
     requests.push(
-      createKnativeService(name, projectName, formData.serverless.scaling, isiName, tag),
+      createKnativeService(
+        name,
+        projectName,
+        formData.serverless.scaling,
```

Comment:
@joshuawilson you can destructure it further, i missed in my PR

```
{serverless: { scaling },
   route,
  } = formData;
```

and then pass `scaling` and `route` in `createKnativeService` here